dist/stat_data.py

```diff
@@ -327,6 +332,22 @@ def __init__(self, name, desc, flags=''):
     CapacityStat('fsync_all_fh_total', 'background fsync file handles considered'),
     CapacityStat('fsync_all_time', 'background fsync time (msecs)', 'no_clear,no_scale'),
 
+    ##########################################
+    # Chunk cache statistics
+    ##########################################
+    ChunkCacheStat('chunk_cache_bytes', 'total bytes'),
```

Comment:
Is this the maximum size from the configuration setting? If so, I suggest naming it `chunk_cache_bytes_max` and using the description `maximum bytes configured`. That makes it both clearer and consistent with the regular cache which has `cache_bytes_max` for a similar purpose.

Also, similar to the regular cache, it seems like a stat for `chunk_cache_bytes_inuse` would be useful with `bytes currently held in the chunk cache`

Reply:
No, this is bytes in use.

Reply:
I will change the name to bytes in use.